demo/demo.py

```diff
@@ -0,0 +1,11 @@
+import pandas as pd
+
+data = [['a','b','c'], ['1','2','3','4']]
+
+def createDataFrame():
+  df = pd.DataFrame(data, columns=['name', 'value', 'key', 'values'])
+  return df
+
+if __name__ == '__main__':
+  df = createDataFrame()
+  print(df)
```

Comment:
**Suggestion:** Remove the unnecessary if __name__ == '__main__': block from the createDataFrame() function.
```suggestion

```